### ISSUE TITLE
BUG: Static Forcefield runs

### ISSUE THREAD

**Issue by JaGeo** [materialsproject/atomate2#397]
**Describe the bug**
Hi @matthewkuner, 

sholdn't those two not be 0 instead of 1?

https://github.com/materialsproject/atomate2/blob/21fd169563026ca5a3e60bac7f6abb7f4d6caac3/src/atomate2/forcefields/jobs.py#L104

https://github.com/materialsproject/atomate2/blob/21fd169563026ca5a3e60bac7f6abb7f4d6caac3/src/atomate2/forcefields/jobs.py#L207

When I run the static runs, a second force output shows up in the optimizer.

**Comment by JaGeo** [comment]
No, this  also does not help. I am sorry.  I will investigate further but there seem to be more than one step for the static runs:

2023-06-23 20:33:24,246 INFO Starting job - CHGNet static 5/54 (4c40e0f5-f156-491e-9c09-7418f0b3e3a5)
```
CHGNet initialized with 400,438 parameters
CHGNet will run on cpu
      Step     Time          Energy         fmax
*Force-consistent energies used in optimization.
FIRE:    0 20:33:40    -7507.195759*       0.3141
FIRE:    1 20:33:56    -7507.220650*       0.1596
FIRE:    2 20:34:12    -7507.238674*       0.0729
```

**Comment by JaGeo** [comment]
Now, I found the workaround in the schema. I think we need to document this better or find a less confusing solution.

**Comment by matthewkuner** [comment]
@JaGeo I noticed issues with trying to get both M3GNet and CHGNet to only run a single step--I think it is an issue with something in ASE itself. I believe @arosen93 submitted an issue there.

How do you propose we document this instead? Just commenting a short blurb about it in the M3GNetStaticMaker and CHGNetStaticMaker?

**Comment by JaGeo** [comment]
Maybe, we can suppress the screen output from the optimizer for the static runs? It's just extremely confusing otherwise.

**Comment by arosen93** [comment]
Some context: https://gitlab.com/ase/ase/-/issues/1258

**Comment by JaGeo** [comment]
Thanks. Yeah. It's probably not expected in ase to use an optimizer if you want to have a static result. I understand, however, that this would make the current implementation much easier.

**Comment by arosen93** [comment]
For sure! Seems like a workaround here is to set nsteps to -1 for the static calc and capture the stdout.

**Comment by matthewkuner** [comment]
I found nsteps=-1 to not always work either :x @arosen93

**Comment by JaGeo** [comment]
Then, we should just capture the stdout then. It took me too long (see thread) to figure out the result is fine, even though an optimization is performed. 🙈

**Comment by arosen93** [comment]
Agreed. For what it's worth, I'll try to open an MR upstream in ASE but might take me a week or two.

**Comment by JaGeo** [comment]
It's somehow fixed, isn't it?

**Comment by arosen93** [comment]
@JaGeo --- the PR was merged in the `master` branch of ASE, yes (but we are still working on a new PyPI release... slowly).

**Comment by JaGeo** [comment]
Thanks. At least, I don't see the confusing output anymore!
